site/public/css/polls.css

```diff
@@ -39,6 +39,17 @@
     cursor: pointer;
 }
 
+.header {
```

Comment:
I think with your new changes we don't use the .header anymore

Reply:
done in [a6df8cc](https://github.com/Submitty/Submitty/pull/10337/commits/a6df8ccacae3e0075b3a3e2eca960331c4b7f316)